tests/unittests/transactionsService.test.ts

```diff
@@ -5,12 +5,12 @@ import { mockedTransaction, mockedGrpc, mockedPaybuttonAddress } from '../mocked
 
 describe('Create services', () => {
   it('Return created transaction', async () => {
-    prismaMock.transaction.create.mockResolvedValue(mockedTransaction)
-    prisma.transaction.create = prismaMock.transaction.create
+    prismaMock.transaction.upsert.mockResolvedValue(mockedTransaction)
+    prisma.transaction.upsert = prismaMock.transaction.upsert
     prismaMock.paybuttonAddress.findMany.mockResolvedValue([mockedPaybuttonAddress])
     prisma.paybuttonAddress.findMany = prismaMock.paybuttonAddress.findMany
 
-    const result = await transactionsService.saveTransaction(
+    const result = await transactionsService.upsertTransaction(
```

Comment:
Since the function of this.. function.. has changed, we should add another test to cover the update case.

Reply:
This test actually covers both cases: since it's an unittest, the DB is ignored and the prisma upsert function is just mocked to return the updated / created object. The object is neither updated or created, of course, because this test is run without the DB being up.